NUnitLite-1.0.0/src/framework/Runner/TextUI.cs

```diff
@@ -112,6 +109,25 @@ public void Execute(string[] args)
             this.commandLineOptions = new CommandLineOptions();
             commandLineOptions.Parse(args);
 
+            if (runner == null) {
+#if MONODROID_TOOLS
+                if (!string.IsNullOrEmpty (commandLineOptions.Android)) {
+                    runner = new Xamarin.AndroidTestAssemblyRunner(commandLineOptions.Android);
```

Comment:
where is this class coming from? Should we just put it into nunitlite? then we could get rid of the #if's.

Reply:
We can't get rid of the #if because we don't want to use a XA / Android specific class for iOS, since that would preclude us from using any feature specific to these.

Reply:
Ok, though I'd still like to see this class in this repo :)

Reply:
I followed the pattern of `Xamarin.BabysitterSupport`.

Reply:
Interesting, didn't know this wasn't in the repo. All good, thanks!